plugins/org.eclipse.elk.alg.force/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -8,27 +8,38 @@ org.eclipse.jdt.core.compiler.debug.lineNumber=generate
 org.eclipse.jdt.core.compiler.debug.localVariable=generate
 org.eclipse.jdt.core.compiler.debug.sourceFile=generate
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enablePreviewFeatures=disabled
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+<<<<<<< HEAD
 org.eclipse.jdt.core.compiler.source=1.8
+=======
+org.eclipse.jdt.core.compiler.problem.reportPreviewFeatures=warning
+org.eclipse.jdt.core.compiler.release=enabled
+org.eclipse.jdt.core.compiler.source=11
+>>>>>>> parent of f5a08b471 (Revert "Purged Java 1.8 from setup and classpath and manifests.")
```

Comment:
Looks like there are some merge conflict markers here that were not resolved [not yet fixed in current master](https://github.com/soerendomroes/elk/blob/d8d5c1b82f0c0579244a174397ac155d57fef2f8/plugins/org.eclipse.elk.alg.force/.settings/org.eclipse.jdt.core.prefs#L13-L19)